pkg/template/api/types.go

```diff
@@ -16,6 +16,10 @@ type Template struct {
 	// Optional: Parameters is an array of Parameters used during the
 	// Template to Config transformation.
 	Parameters []Parameter `json:"parameters,omitempty"`
+
+	// Optional: ObjectLabels is a set of labels that are applied to every
+	// object during the Template to Config transformation
+	ObjectLabels map[string]string `json:"objectlabels,omitempty"`
```

Comment:
Yes, they will.  There may be a scenario where the default labels are those of the object (if you do something specific).

----- Original Message -----
> > @@ -16,6 +16,10 @@ type Template struct {
> >  	// Optional: Parameters is an array of Parameters used during the
> >  	// Template to Config transformation.
> >  	Parameters []Parameter `json:"parameters,omitempty"`
> > +
> > +	// Optional: ObjectLabels is a set of labels that are applied to every
> > +	// object during the Template to Config transformation
> > +	ObjectLabels map[string]string `json:"objectlabels,omitempty"`
> 
> @mfojtik that seems slightly abusive, particularly once we have a template
> repository and might want templates to actually have their own labels for
> their own purposes.
> 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/982/files#r24751605